_posts/2017/2017-03-21-continuous-documentation-with-ansible.md

```diff
@@ -0,0 +1,220 @@
+---
+layout: post
+title: "Continuous Documentation with Ansible"
+date: "2017-03-21 06:39:20"
+image: blog-header/scrum-board.jpg
+categories: tech-stories agile devops docker jenkins ansible
+authors: ["Thomas H."]
+---
+
+## Introduction
+In recent years there has been a [steady trend towards cloud based deployments](http://www.softwareadvice.com/buyerview/deployment-preference-report-2014/) of applications and services.  
+This increases the distributed nature of the systems we develop, which in turn raises the complexity and effort
+required to develop and manage (i.e. provision resources, build, test, configure, deploy and monitor) them.  
+This is especially true, as during this transition phase the environment is heterogeneous and different methods of building and deploying software are combined (e.g. RPM packages and Docker containers).
+
+At the same time, delivery cycles are expected to shorten because of the [opportunity cost of not releasing software](http://robertbaillie.blogspot.de/2014/10/the-opportunity-cost-of-delaying.html)
+and the [Agile principles](http://agilemanifesto.org/principles.html) of customer collaboration and early feedback.
+
+This post will outline how [Ansible](https://www.ansible.com/) can help in approaching these challenges.
+It also includes a short discussion on the relation of Ansible to other tools supporting continuous delivery and
+scalability of the development process, [Docker](https://www.docker.com/) and [Jenkins](https://jenkins.io/).
+
+## Documentation vs. Automation
+<img src="/assets/images/blog/blog-dilbert-agile.gif" alt="Dilbert comic agile" style="width: 100%;"/>
```

Comment:
I'm not sure if we're allowed to use this unless we mention where it's from.
I recommend to remove it.